src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Appointment.kt

```diff
@@ -46,6 +47,8 @@ data class Appointment(
   @PrimaryKeyJoinColumn
   var appointmentDelivery: AppointmentDelivery? = null,
 
+  @ManyToOne(fetch = FetchType.LAZY) var referral: Referral,
```

Comment:
Originally the choice was a referral id - I like the idea of the referral but what was the reason for the change